docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -5,45 +5,40 @@
 If you already installed the supported version `@zowe-v1-lts`, switch versions to try this feature. Daemon mode will be included in the next major Zowe release, V2.0.0-LTS. You can also [install the @next release of Zowe CLI](cli-install-cli-next.md#install-zowe-cli-from-a-download).
 
 **Table of Contents:**
+
 - [Feature overview](#feature-overview)
-- [Install the zowex native executable](#install-the-zowex-native-executable)
-- [Starting Zowe in daemon mode](#starting-zowe-in-daemon-mode)
+- [Install the zowe native executable](#install-the-zowe-native-executable)
+- [Running zowe commands with the zowe executable](#running-zowe-commands-with-zowe-executable)
 
 ## Feature overview
+
 Daemon Mode significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process (daemon).
 
-## Install the zowex native executable
-You can download and install the zowex native executable from Zowe Downloads (preferred) or from the Zowe Github repository. To install the zowex native executable from Zowe Downloads, see [Install Zowe CLI from a download](cli-install-cli-next.md#install-zowe-cli-from-a-download).
+## Install the zowe native executable
+
+You can download and install the zowe native executable from Zowe Downloads (preferred) or from the Zowe Github repository. To install the zowe native executable from Zowe Downloads, see [Install Zowe CLI from a download](cli-install-cli-next.md#install-zowe-cli-from-a-download).
+
+To download and install the zowe native executable from the Zowe Github repository:
 
-To download and install the zowex native executable from the Zowe Github repository:
 1. Navigate to the Zowe CLI GitHub [repository](https://github.com/zowe/zowe-cli)
 2. Under the **Release** section, click **Native Client Release**.
    The Download page is displayed.
 3. Click the link to download the executable (.tgz file) for your operating system.
-4. Unzip the zowex .tgz file and place the zowex (or zowex.exe) file into a directory which occurs on your PATH earlier than the directory containing the NodeJS zowe command.
-
+4. Unzip the zowe .tgz file and place the zowe (or zowe.exe) file into a directory which occurs on your PATH earlier than the directory containing the NodeJS zowe command.
```

Comment:
I'd like to add a note (for Mac Users) alongside these steps to indicate that people may (or may not) need to accept GitHub as an unidentified developer.
More details (and link with steps) can be found in the issue:
- https://github.com/zowe/zowe-cli/issues/1193